### PR TITLE
OOP School Library: Create Initial Classes

### PR DESCRIPTION
**Pull request by BenjaminSemah**
### In this branch, I have done the following

✅ created a class `Person`
✅ created a class `Student`
✅ created a class `Teacher`
✅ updated the ReadMe file
✅ fixed all errors spotted by linters

### CODE REVIEW

File: person.rb [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+    @name = name
+    @age = age
+    @parent_permission = parent_permission
+  end
+
+  attr_reader :id
+  attr_accessor :name, :age
+
+  def can_use_services?
+    of_age? || @parent_permission ? true : false
```
Comment (mikethreels) [inline person.rb:13]: - [x] [OPTIONAL] you don't need to include the ternary operator in this case, the below will also return a boolean value:
```
def can_use_services?
  of_age? || @parent_permission
end
```

**Review by mikethreels — APPROVED** [review]
Hi Benjamin, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: person.rb (hunk 1 above)
Comment (BenjaminSemah) [reply person.rb:13]: O okay. Noted.
Thanks.

**Comment by BenjaminSemah** [comment]
Thanks for the review @mikethreels

**Pull request merged**